src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -53,16 +53,17 @@ public void locateComponentsForOfflineDetectorScan(BdioResult bdio, File scanOut
 
     private void runComponentLocator(List<Component> componentsList, File scanOutputFolder, File projectSrcDir) throws ComponentLocatorException {
         Input componentLocatorInput = generateComponentLocatorInput(componentsList, projectSrcDir);
+        String outputFilepath = scanOutputFolder + "/" + DETECT_OUTPUT_FILE_NAME;
+
         logger.info(ReportConstants.RUN_SEPARATOR);
-        logger.info("Running Component Locator.");
-        String outputFile = scanOutputFolder + "/" + DETECT_OUTPUT_FILE_NAME;
-        int status = ComponentLocator.locateComponents(componentLocatorInput, outputFile);
+        int status = ComponentLocator.locateComponents(componentLocatorInput, outputFilepath);
         if (status != 0) {
```

Comment:
Dummy lib will return 1 by default as of v1.0.7

Reply:
Will real library return 1 for any other reason? If yes, can we return a different number from dummy library for clarity.

Reply:
- The dummy lib will always return 1
- The real lib will return 1 in all cases where the execution was unsuccessful 

In the case of dummy lib, the Detect console output looks like: 
![Screenshot 2023-07-24 at 2 29 23 PM](https://github.com/blackducksoftware/synopsys-detect/assets/121134650/2b107bce-3cc5-4b44-8f57-95917166bfb6)
![Screenshot 2023-07-24 at 2 29 34 PM](https://github.com/blackducksoftware/synopsys-detect/assets/121134650/c4d14d8c-1c1f-4c13-9763-3a204d76297e)


In the case of real lib, the Detect console output looks like: 
**When successful:** 
![Screenshot 2023-07-24 at 2 33 48 PM](https://github.com/blackducksoftware/synopsys-detect/assets/121134650/0f70eb27-aa2f-4d63-9787-067945fdcb61)
**When unsuccessful:** 
![Screenshot 2023-07-24 at 2 37 05 PM](https://github.com/blackducksoftware/synopsys-detect/assets/121134650/ae762bd7-78ca-4bbc-8cb9-6e052caa27b4)
![Screenshot 2023-07-24 at 2 37 11 PM](https://github.com/blackducksoftware/synopsys-detect/assets/121134650/e3092d9a-2eb0-46fc-9eb1-b8f53a8ff3c3)


I suggest we keep the return statuses as they are, since right now the really does not give any more detailed info than it does above. As in, it's not like we have different return statuses indicating different failures (yet .. post mVP this would be a good add of course). For now, if we changed the return status of real and dummy to be diff, it would add no value but just an additional if statement here.

Reply:
I am thinking of customer support issues. Should get get a report that customer is getting a non-zero response, and if it turns out to be 42, then we could know that the customer built Detect and is trying the feature.
If it is 1, then we would know that it is CLL and there was a failure.